server/main.go

```diff
@@ -0,0 +1,79 @@
+package main
+
+import (
+	"bufio"
+	"context"
+	"log"
+	"net"
+	"os"
+	pb "server/file_manager"
+
+	"google.golang.org/grpc"
+)
+
+type server struct {
+	pb.UnimplementedFileManagerServer
+}
+
+func (s *server) PutFile(ctx context.Context, in *pb.PutFileRequest) (*pb.PutFileResponse, error) {
+	var fileName = in.GetFileName()
+	var fileContext = in.GetFileContext()
+
+	log.Printf("Received fileName and fileContext as : %v", fileName+" | "+fileContext)
+
+	writeFile(fileName, fileContext)
+
+	// writeFile 에서 string을 리턴하고, 현재 함수에서 return값을 받아와서 PutFileResponse로 리턴할 수 없으면 결과 어떻게 로그찍음??
+	return &pb.PutFileResponse{Result: "File saved successfully"}, nil
+}
+
+func writeFile(fileName string, fileContext string) {
+	file, err := os.Create("/Users/juwon/Desktop/go-storage/" + fileName + ".txt")
+	if err != nil {
+		log.Fatal(err)
```

Comment:
```suggestion
		return err
```
It's the best practice to just return an error to the top, so that the caller can handle the error.
In Go and any other language, it's recommended to throw errors to the top-most layer, so that we can write a single log at the top-most layer. If not, we would write multiple logs for the same error (at each layer).